prclz/etl/_download.py

```diff
@@ -26,16 +26,14 @@ def build_data_dir(root: str, additional: Optional[Sequence[str]] = None, permis
         for path in data_paths.values():
             dir_path.mkdir(parents=True, exist_ok=True)
     else:
-        info('Downloading will create the following repositories:')
+        info('Downloading will create the following repositories if they do not already exist:')
         for dir_path in data_paths.values():
-            info(f'{dir_path}')
+            info(f'{dir_path.resolve()}')
         user_response = input('Do you wish to proceed with download to the specified folder? Please enter "y" or "n".')
         while user_response.lower() not in ['y', 'n']:
-            user_repsonse = input('Please enter "y" or "n".')
+            user_repsonse = input('Please enter "y" or "n"')
```

Comment:
typo

Reply:
might be worth doing a run with `mypy` or other linters to catch this sort of thing